components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -1148,4 +1168,8 @@ get_var_dimnames (const FieldLayout& layout) const
   return dims;
 }
 
+// Instantiate the static member var
+AtmosphereOutput::strmap_t<AtmosphereOutput::diag_ptr_type>
+AtmosphereOutput::m_diag_repo;
```

Comment:
It's unclear to me where we add anything to `m_diag_repo`.  A quick search of the term `m_diag_repo` doesn't show any case of setting a value, just accessing a value.  Am I missing something?

Reply:
It's in the previous mod. Namely, the line
```cpp
   auto& diag = m_diag_repo[name];
```